README.md

```diff
@@ -21,6 +21,35 @@ This project hosts the JavaScript versions of [Realm](https://realm.io/). Curren
 
 Please see the detailed instructions in our docs to use [Realm JavaScript for Node.js](https://docs.mongodb.com/realm/sdk/node/) and [Realm JavaScript for React Native](https://docs.mongodb.com/realm/sdk/react-native/). Please notice that currently only Node.js version 13 or later is supported. For React Native users, we have a [compatibility matrix](COMPATIBILITY.md) showing which versions are supported.
```

Comment:
Docs URL has changed. New ones are https://www.mongodb.com/docs/realm/sdk/node/ and https://www.mongodb.com/docs/realm/sdk/react-native/. I added https://github.com/realm/realm-js/pull/5405